src/shared/types/User.ts

```diff
@@ -1,4 +1,5 @@
 export interface User {
+  role: string;
```

Comment:
Please add Union type for "role". It will help us in the future.
"admin"
"manager"
"auth_user"
"guest_user"

Reply:
complete